numpy/tests/test_ctypeslib.py

```diff
@@ -8,7 +8,14 @@
 from numpy.testing import TestCase, run_module_suite, dec
 
 try:
-    cdll = load_library('multiarray', np.core.multiarray.__file__)
+    cdll = None
+    if hasattr(sys, 'gettotalrefcount'):
+        try:
+            cdll = load_library('multiarray_d', np.core.multiarray.__file__)
+        except OSError:
```

Comment:
I suspect the fix should be in `load_library`, which already tries two variant file name extensions from `get_shared_lib_extension`. Remind me of where exactly the `_d` comes from?